api/types.go

```diff
@@ -553,6 +553,9 @@ type CreateRequest struct {
 	Renderer string `json:"renderer,omitempty"`
 	Parser   string `json:"parser,omitempty"`
 
+	// MinVersion is the minimum version of Ollama required by the model.
+	MinVersion string `json:"min_version,omitempty"`
```

Comment:
Just for future proofing, do you want to add a MaxVersion as a way of deprecating models? We could add that in a follow up though.